package.json

```diff
@@ -6,22 +6,23 @@
   "bugs": "https://github.com/forcedotcom/cli/issues",
   "dependencies": {
```

Comment:
Since I was pulling in `@salesforce/lwc-dev-mobile-core` I also decided to update the rest of the dependencies to the latest version.